tests/execute/result/test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "run=\$(mktemp -d)" 0 "Create run directory"
+        rlRun "pushd data"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+
+    rlPhaseStartTest "Running the test"
+        rlRun "tmt run -a --scratch --id ${run} provision --how local" 2 "Main test"
+        results="${run}/plans/default/execute/results.yaml"
+
+        rlRun "cat ${results}" 0 "Show the results.yaml"
+        rlRun -s "cat ${results} | python3 ../parse_result.py" 0 "Parse the results.yaml"
```

Comment:
Instead of parsing the `results.yaml` then printing it and then grepping it and grepping once again, wouldn't it be better to just select the appropriate test and check the output in verbose mode?

    tmt run -v test --name xfail-fail 

This gives:

    report
        how: display
            pass /test/xfail-fail (original_result: fail)
        summary: 1 test passed

Where both `result` and `note` can be checked.

Reply:
I'll rework it like that.

Reply:
Thanks for improving the test, looks much better now!